src/shipit_code_coverage/tests/fixtures/linux_test_task_artifacts.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Let's get a real response from taskcluster instead of crafting a fake one. This way we also test that `download` only downloads the needed artifacts and not all artifacts.

Reply:
But this json has 2 artifacts and only one needs to be downloaded. The assert in my test `assert mocked_download_artifact.call_count == 1` shows that it only tries to download one artifact and it is the right one as tested by the assert down this one.

Reply:
Right, I hadn't noticed there are two artifacts. I would use a real reply anyway.

Reply:
I got it too :)
After I had added the parent_dir, it can found the path for the artifact that it tries to download, so that path is returned and no request needs to be done. Right?

Reply:
No request is done anyway, since you are mocking the `taskcluster` calls.
I'm not sure the parent_dir argument is actually needed. Since you are mocking the functions, nothing is actually downloaded.

Reply:
We should copy an actual reply here instead of creating one ourselves. For example, you can use this: https://queue.taskcluster.net/v1/task/FdAEmN1DQzexvnbQruoYbw/artifacts.

Reply:
aah ok, then

Reply:
The file should only contain exactly what https://queue.taskcluster.net/v1/task/FdAEmN1DQzexvnbQruoYbw/artifacts contains.